From 459683464cfa96b7a49a13297f4ce30efc1c7b4c Mon Sep 17 00:00:00 2001
From: Iain Sandoe <iain@sandoe.co.uk>
Date: Fri, 10 Dec 2021 21:20:38 +0000
Subject: [PATCH] libgcc, Darwin: Update darwin10 unwinder shim dependencies.

We include libgcc_tm.h to provide a prototype for this shim
so add that to the make dependencies.

Signed-off-by: Iain Sandoe <iain@sandoe.co.uk>

libgcc/ChangeLog:

	* config/t-darwin: Add libgcc_tm.h to the dependencies
	for darwin10-unwind-find-enc-func.

(cherry picked from commit b504917e43b9a559c9ac779e08784ad412125f2e)
---
 libgcc/config/t-darwin | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git libgcc/config/t-darwin libgcc/config/t-darwin
index 7c0f1a64ff4..2aab1273287 100644
--- libgcc/config/t-darwin
+++ libgcc/config/t-darwin
@@ -27,7 +27,7 @@ libemutls_w.a: emutls_s.o
 # Patch to __Unwind_Find_Enclosing_Function for Darwin10.
 # This has to be built for 10.6, even if the toolchain will not target that
 # version
-d10-uwfef.o: $(srcdir)/config/darwin10-unwind-find-enc-func.c
+d10-uwfef.o: $(srcdir)/config/darwin10-unwind-find-enc-func.c libgcc_tm.h
 	$(crt_compile) -mmacosx-version-min=10.6 -c $<
 
 # Using this crt as a library means that it will not be added to an exe
-- 
2.40.1

